tor/core/user_interaction.py

```diff
@@ -185,6 +204,11 @@ def process_done(post, config, override=False, alt_text_trigger=False):
                     f'Attempted to mark post {top_parent.fullname} '
                     f'as done... hit ClientException.'
                 )
+
+            config.redis.hdel(
```

Comment:
In the current configuration, this will fire if someone else attempts to complete the post.